sdk/python/kfp/compiler/pipeline_spec_builder.py

```diff
@@ -2063,12 +2063,24 @@ def write_pipeline_spec_to_file(
 
 def _merge_pipeline_config(pipelineConfig: pipeline_config.PipelineConfig,
                            platformSpec: pipeline_spec_pb2.PlatformSpec):
-    # TODO: add pipeline config options (ttl, semaphore, etc.) to the dict
-    # json_format.ParseDict(
-    #     {'pipelineConfig': {
-    #         '<some pipeline config option>': pipelineConfig.<get that value>,
-    #     }}, platformSpec.platforms['kubernetes'])
+    workspace = pipelineConfig.workspace
```

Comment:
Do you need to guard against `pipelineConfig.workspace` being `None`/not set below?